src/app/faq/faq.controller.js

```diff
@@ -9,14 +9,78 @@
   function FaqController() {
 
     var vm = this;
-
-    vm.menuStatus = [
-      { isOpen : true     },
-      { isOpen : false    },
-      { isOpen : false    },
-      { isOpen : false    },
-      { isOpen : false    },
-      { isOpen : false    }
+    
+    vm.faq = [
+      {
+        category : "General",
```

Comment:
The category should be pulled out and all the FAQs should be nested within a category. Also, the name of the category should appear on the page as an H1 heading.